resources/views/admin/emails/main.blade.php

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Put all emails in `emails` directory. Make all subfolders there.
So it will look like `resources/views/emails/admin/main.blade.php`